app/controllers/users/registrations_controller.rb

```diff
@@ -28,6 +28,7 @@ def create
 
     ActiveRecord::Base.transaction do
       resource.save!
+      @school.induction_coordinator_profiles.destroy_all
```

Comment:
I'm not sure what we need to do with the coordinator who hasn't confirmed their email. Here I'm just removing them from the school. They remain induction coordinators but they no longer belong to a school.  Not sure what the best thing to do should be. We could just destroy the user record and profile.

Reply:
Who do you think would be the best person to answer the question - the designers? BA's - It's a valid question and it seems like it is not about technical implementation but the product functionality